### PR TITLE
move meeting template toggle to menu and add confirmation

### LINKED ISSUE #1306: Change the appearance and settings of meeting templates
`.../committees/...`

**Problem:**
The star icon "Template meeting" triggers to click on it. Once a meeting is set as a template, it is publicly visible to all committee administrators. So even those who should not have access to it.

**Solution:**

- The star icon is removed.
- A checkbox (deactivated by default) is added to the three-dots-menu of the individual meeting: `Public template`

**When the checkbox is clicked to activate, a dialog will appear (structure follows this dialog):** 

Screenshot 2022-06-22 at 17 38 38

1st line (in bold): Should this meeting really be made available as a public template? Meeting templates and the data they contain are publicly viewable by all committee administrators.``
2nd line: [Meeting name]
Options: `OK` `Cancel`

1. Zeile (fett geschrieben): Soll diese Veranstaltung wirklich als öffentliche Vorlage zur Verfügung gestellt werden? Veranstaltungsvorlagen und darin enthaltene Daten sind öffentlich für alle Gremienverwalter einsehbar.
2. Zeile: [Name der Veranstaltung]
Optionen: `OK` `Abbrechen`

**When the checkbox is clicked to deactivate, a dialog will appear (structure follows dialog above):**

1st line (written in bold): Do you really want to stop sharing this meeting as a public template?
2nd line: [Meeting name]
Options: `OK` `Cancel``

1. Zeile (fett geschrieben): Soll dieses Meeting wirklich nicht mehr als öffentliche Vorlage zur Verfügung gestellt werden?
2. Zeile: [Name der Veranstaltung]
Optionen: `OK` `Abbrechen`

### PR DESCRIPTION
**Pull request by bastianjoel**
resolves #1306 

TODO:

- [ ] German translation

### CODE REVIEW

File: client/src/app/site/pages/organization/pages/committees/modules/committee-meeting-preview/committee-meeting-preview.component.ts [hunk 1]
```diff
@@ -100,10 +100,26 @@ export class CommitteeMeetingPreviewComponent {
...
+            );
+
+            const confirmed = await this.promptService.open(title, content);
+            if (confirmed) {
+                await this.meetingRepo.update({ set_as_template: false }, { meeting: this.meeting });
+            }
         } else {
-            await this.meetingRepo.update({ set_as_template: true }, { meeting: this.meeting });
+            const title = this.translate.instant(
+                `Should this meeting really be made available as a public template? Meeting templates and the data they contain are publicly viewable by all committee administrators.`
```
Comment (luisa-beerboom) [inline client/src/app/site/pages/organization/pages/committees/modules/committee-meeting-preview/committee-meeting-preview.component.ts:116]: This is probably too long for a title, maybe you should put the explanation part in the content.

**Review by luisa-beerboom — CHANGES_REQUESTED** [review]

File: client/src/app/site/pages/organization/pages/committees/modules/committee-meeting-preview/committee-meeting-preview.component.ts (hunk 1 above)
Comment (bastianjoel) [reply client/src/app/site/pages/organization/pages/committees/modules/committee-meeting-preview/committee-meeting-preview.component.ts]: Bildschirmfoto_2022-08-17_13-29-58

**Review by luisa-beerboom — APPROVED** [review]

**Review by emanuelschuetze — APPROVED** [review]
Works good. - But I think we should show the star icon (in the meeting title bar) if template is activated. Otherwise you cannot see the template state without clicking the dot menu.
@alveolus What do you mean?

**Comment by emanuelschuetze** [comment]
@bastianjoel 
As discussed with @alveolus: Please add star icon to meeting title bar if template state is active. And keep title bar empty (no star) if template state is inactive.

**Review by luisa-beerboom — APPROVED** [review]

**Review by emanuelschuetze — APPROVED** [review]

**Pull request merged**